src/domains/marketing/Features.js

```diff
@@ -57,8 +60,13 @@ export function Features({ features = FOLLOWS_FILTERS }) {
                 Made by
               </h2>
               <a
-                href="https://queen.raae.codes/?utm_campaign=prune+your+follows&utm_source=app&utm_medium=features"
                 className="group mt-1.5 block max-w-xs"
+                onClick={() =>
```

Comment:
suggestion(blocking): this is no longer a link, so it will not take you to queen.raae.codes. The href needs to be put back in place, and then use trackGoal same way as above. But with correct goal key.

Reply:
Oh, ok I'll do that tomorrow then, saw this a little late 
😬